hardware/blink1/package.json

```diff
@@ -13,7 +13,7 @@
     "keywords": [ "node-red", "blink1" ],
     "node-red"      : {
         "nodes"     : {
-            "blink1": "77-blink1.js"
+            "blink1": "blink1.js"
```

Comment:
can't change this - needs to be as-was.

Reply:
You said in Slack to change the file names if I want to. Now you are saying I shouldn't. What is it?

Reply:
I reread what you wrote - I misunderstood the filename comment. Will revert to 77-*